qemu/tests/cfg/create_large_img.cfg

```diff
@@ -1,7 +1,6 @@
-- create_large_raw_img:
-    only raw
+- create_large_img:
```

Comment:
test case name changed, we may also need to change test loop. Please have a check

Reply:
Thank you for your reminder, I will update test loop after this patch merged.